unit_tests.py

```diff
@@ -21,6 +21,8 @@
 # --collect-only, --co  only collect tests, don't execute them.
 # pytest tests.py --co
 
+# A "test_" prefix in classes and methods is needed to make a test discoverable by pytest
+
```

Comment:
Oh and one final tiny request: could you please add the `-v` option in this file as well?